### DISCUSSION TITLE
Unable to Rename Files While Downloading in Latest Version; "This action can't be completed because the file is open in ffmpeg"

### DISCUSSION THREAD

**Discussion by jeff15110168** [aclap-dev/video-downloadhelper#283] [Help and Q&A]
### Your question

I use Video DownloadHelper mostly because of the ability to download files and rename them before the download is finished. I typically will have something like 50-100 tabs open, right click -> download .mp4 -> copy paste the title within the tab to the file created and then close tab. I will simultaneously be downloading 10-20 files at a time.

This allows me very quickly know which file is what, and not have to wait until the download is done, then cross-check the file video lengths with the open tab to know what title belongs to which video file. I know the auto-name functionality can partially solve this but it'd be great if I could revert to the old behavior that just lets me rename the file while-being-downloaded as a default.

If i try to rename a file within file explorer while being downloaded, i get the error "This action can't be completed because the file is open in ffmpeg". I'm not doing any transcoding.

Separately - the new version that auto-updated, the videos on certain sites only show up when i click the icon in the top right, but not when i right click within the website. What could be causing this and could fix this behavior so the video that i want to download (on the opened tab) appears when i right click on the page?

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\DEFAULT",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by jcfrog** [comment]
Hi,
If you select Download as default action you should be prompted to change file name
<img width="515" alt="Screenshot 2024-03-12 at 08 45 28" src="https://github.com/aclap-dev/video-downloadhelper/assets/368367/5ce42cfc-9e15-43c5-a1d2-099acfa56575">

**Comment by jcfrog** [comment]
for the right click issue, can you share a public link to reproduce it?

**Reply by jeff15110168** [reply]
it's not a public link (private in nature). But this is related to the functionality changing where i could rename a file while still being downloaded while now it seems like the file is getting locked up by ffmpeg while downloading
